src/hooks/useConnectorImagesSpec.ts

```diff
@@ -1,57 +1,61 @@
-import { type NewCaptureState } from 'components/capture/creation/Reducer';
+import {
+    ConnectorImagesSpecResponse,
+    connectorsEndpoint,
+} from 'endpoints/connectors';
+import { useAsync } from 'hooks/useAsync';
 import JsonRefs from 'json-refs';
-import { useEffect, useState } from 'react';
-import axios, { withAxios } from 'services/axios';
-import { type BaseHook } from 'types';
-
-interface ConnectorImagesService extends BaseHook {
-    data: {
-        specSchema: any;
-        links: Pick<NewCaptureState['links'], 'discovery' | 'documentation'>;
+import { useEffect } from 'react';
+
+interface Data {
+    endpointSchema: ConnectorImagesSpecResponse['data']['attributes']['endpointSpecSchema'];
+    links: {
+        discovery: ConnectorImagesSpecResponse['data']['links']['discovery'];
+        documentation: ConnectorImagesSpecResponse['data']['attributes']['documentationURL'];
     };
 }
```

Comment:
We can see how this goes. This hook filters out data and returns specific portions of it. All the other hooks just pass along all of it. I think eventually we'll figure out which way is best.

Also... handling the typing like this is kind of silly since it does not really add any benefits other than helping "document" what values were intended for this object.